index.html

```diff
@@ -111,6 +112,15 @@
 				return [[minX, minY], [maxX, maxY]];
 			}
 
+			function fillpopup(data){
+				var html = "";
+				html = html + "<span class='varname'>District Name: </span> <span class='attribute'>" + data.SchDistNam + "</span>";
+				html = html + "<br>"
+				html = html + "<span class='varname'>Increase per ADA</span> <span class='attribute'>" + data.EstADAIn +"</span>";
+				return html;
+				//this will return the string to the calling function
+			}
+
```

Comment:
This function is unchanged; I just had to move it upstream so it would be defined before the first thing that calls it.